python/src/trezorlib/messages.py

```
@@ -3890,30 +3898,39 @@
     FIELDS = {
         1: protobuf.Field("ecdsa_curve_name", "string", repeated=False, required=False),
         2: protobuf.Field("paths", "Path", repeated=True, required=False),
+        3: protobuf.Field("include_node", "bool", repeated=False, required=False),
     }
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)